game/outfits.rpy

```diff
@@ -0,0 +1,304 @@
+        unlocked=True,
+        clothes="casual",
+        hairstyle="ponytail",
+        accessory="hairbands/white"
+    )
+
+    DEFAULT_OUTFIT_CASUAL_WEEKEND = Outfit(
+        display_name="Casual clothes",
+        reference_name="jn_casual_weekend",
+        unlocked=True,
+        clothes="casual",
+        hairstyle="bun",
+        accessory="hairbands/red"
+    )
+
+    DEFAULT_OUTFIT_CASUAL_WEEKEND_ALT = Outfit(
+        display_name="Casual clothes",
+        reference_name="jn_casual_weekend_alt",
+        unlocked=True,
+        clothes="casual",
+        hairstyle="messy_bun",
+        accessory="hairbands/white"
+    )
+
+    DEFAULT_OUTFIT_NIGHT = Outfit(
+        display_name="Pyjamas",
+        reference_name="jn_pajamas_night",
+        unlocked=True,
+        clothes="star_pajamas",
+        hairstyle="down",
+        accessory="hairbands/red"
+    )
+
+    DEFAULT_OUTFIT_MORNING = Outfit(
+        display_name="Pyjamas",
+        reference_name="jn_pajamas_morning",
+        unlocked=True,
+        clothes="star_pajamas",
+        hairstyle="bedhead",
+        accessory="hairbands/red"
+    )
+
+    DEFAULT_OUTFIT_MORNING_ALT = Outfit(
+        display_name="Pyjamas",
+        reference_name="jn_pajamas_morning_alt",
+        unlocked=True,
+        clothes="star_pajamas",
+        hairstyle="bedhead",
+        accessory="hairbands/green"
+    )
+
+    DEFAULT_OUTFIT_CHRISTMAS = Outfit(
+        display_name="Natsu Claus",
+        reference_name="jn_natsu_claus",
+        unlocked=True,
+        clothes="casual",
+        hairstyle="down",
+        accessory="hairbands/green",
+        headgear="natsu_claus_hat"
+    )
+
+    # Default outfit schedules
+    DEFAULT_OUTFIT_SCHEDULE_WEEKDAY_HIGH_AFFINITY = {
+        utils.TIME_BLOCK_EARLY_MORNING: DEFAULT_OUTFIT_MORNING_ALT if _use_alt_outfit else DEFAULT_OUTFIT_MORNING,
+        utils.TIME_BLOCK_MID_MORNING: DEFAULT_OUTFIT_UNIFORM,
+        utils.TIME_BLOCK_LATE_MORNING: DEFAULT_OUTFIT_UNIFORM,
+        utils.TIME_BLOCK_AFTERNOON: DEFAULT_OUTFIT_UNIFORM,
+        utils.TIME_BLOCK_EVENING: DEFAULT_OUTFIT_CASUAL_WEEKDAY_ALT if _use_alt_outfit else DEFAULT_OUTFIT_CASUAL_WEEKDAY,
+        utils.TIME_BLOCK_NIGHT: DEFAULT_OUTFIT_NIGHT
+    }
+
+    DEFAULT_OUTFIT_SCHEDULE_WEEKEND_HIGH_AFFINITY = {
+        utils.TIME_BLOCK_EARLY_MORNING: DEFAULT_OUTFIT_MORNING_ALT if _use_alt_outfit else DEFAULT_OUTFIT_MORNING,
+        utils.TIME_BLOCK_MID_MORNING: DEFAULT_OUTFIT_MORNING_ALT if _use_alt_outfit else DEFAULT_OUTFIT_MORNING,
+        utils.TIME_BLOCK_LATE_MORNING: DEFAULT_OUTFIT_MORNING_ALT if _use_alt_outfit else DEFAULT_OUTFIT_MORNING,
+        utils.TIME_BLOCK_AFTERNOON: DEFAULT_OUTFIT_CASUAL_WEEKEND_ALT if _use_alt_outfit else DEFAULT_OUTFIT_CASUAL_WEEKEND,
+        utils.TIME_BLOCK_EVENING: DEFAULT_OUTFIT_CASUAL_WEEKEND_ALT if _use_alt_outfit else DEFAULT_OUTFIT_CASUAL_WEEKEND,
+        utils.TIME_BLOCK_NIGHT:DEFAULT_OUTFIT_NIGHT
+    }
```

Comment:
I see what you're doing here but I think this can be executed better, noting the `class` idea for this instead, we should also optimize the conditions here. We're evaluating the same one numerous times when we only really need to once for most of these

Reply:
As before - not sure on the value of doing this now if outfits require a rework anyway